fast/stages/0-bootstrap/billing.tf

```diff
@@ -74,12 +74,3 @@ resource "google_billing_account_iam_member" "billing_ext_admin" {
   role               = "roles/billing.admin"
   member             = each.key
 }
-
-resource "google_billing_account_iam_member" "billing_ext_cost_manager" {
```

Comment:
You don't assign this anywhere else. Is this no needed if you have `billing.admin`?

Reply:
It overlaps with billing admin, yes